src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -38,6 +62,13 @@ def get(self, request, project):
             if not can_create_tickets and node.id in TICKET_ACTIONS:
                 continue
 
+            # Create as many of these nodes as there are Sentry Apps with UI Alert Rule components
+            if node.id == "sentry.rules.actions.notify_event_sentry_app.NotifyEventSentryAppAction":
+                custom_actions = node.get_custom_actions(project)
+                if len(custom_actions) != 0:
```

Comment:
`custom_actions` is a list so it'd be safe to do 
```python
if custom_actions:
```
or 
```python
if len(custom_actions):
```

Reply:
Will change in https://github.com/getsentry/sentry/pull/28659